resources/qml/WelcomePages/WhatsNewContent.qml

```diff
@@ -21,39 +25,170 @@ Item
         anchors.top: parent.top
         anchors.horizontalCenter: parent.horizontalCenter
         horizontalAlignment: Text.AlignHCenter
-        text: catalog.i18nc("@label", "What's new in Ultimaker Cura")
+        text: catalog.i18nc("@label", "What's New")
         color: UM.Theme.getColor("primary_button")
         font: UM.Theme.getFont("huge")
         renderType: Text.NativeRendering
     }
 
-    Cura.ScrollableTextArea
+    Item
     {
-        id: whatsNewTextArea
-
+        id: topSpacer
         anchors.top: titleLabel.bottom
-        anchors.bottom: getStartedButton.top
-        anchors.topMargin: UM.Theme.getSize("wide_margin").height
-        anchors.bottomMargin: UM.Theme.getSize("wide_margin").height
-        anchors.left: parent.left
-        anchors.right: parent.right
+        height: UM.Theme.getSize("default_margin").height
+        width: UM.Theme.getSize("default_margin").width
+    }
+
+    Rectangle
+    {
+        anchors
+        {
+            top: topSpacer.bottom
+            bottom: whatsNewDots.top
+            horizontalCenter: parent.horizontalCenter
+            margins: UM.Theme.getSize("default_margin").width
+        }
+
+        height: parent.height - (UM.Theme.getSize("default_margin").height * 2)
```

Comment:
Why not use anchors? This way feels a bit prone to errors.

Reply:
Managed to decomplicate the situation a little bit (I think). Try review again? :-)